src-tests/data/test6b.netrc

```diff
@@ -0,0 +1 @@
+default login abc password ""multi-word password""
```

Comment:
Should this be `"multi-word password"`, like a string in double-quotes?
Now it is _double double-quotes_...